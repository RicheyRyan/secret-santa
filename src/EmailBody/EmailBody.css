.email-body{
  border: 1px dotted #999;
  padding: 2em;
  margin-bottom: 2em;
}
