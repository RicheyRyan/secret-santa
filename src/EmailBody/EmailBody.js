import React from 'react';
import './EmailBody.css';
import { getUsersName } from '../Utils/user.js';

export default props =>
  <div className="email-body">
    <p>Hi {props.user.name.first},</p>
    <p>Welcome to Secret Santa!</p>
    <p>
      You've been chosen to get a gift for {getUsersName(props.matchedUser)}.
    </p>
    <p>Make sure you get them something nice.</p>
    <p>All the best,</p>
    <p>The Secret Santa Team</p>
  </div>;
