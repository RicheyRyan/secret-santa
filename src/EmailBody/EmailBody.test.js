import React from 'react';
import renderer from 'react-test-renderer';
import EmailBody from './EmailBody.js';

const currentUser = {
  name: {
    first: 'Test',
    last: 'Name',
  },
};
const matchedUser = {
  name: {
    first: 'Test',
    last: 'Name',
  },
};

it('Cta component matches snapshot', () => {
  const component = renderer.create(
    <EmailBody user={currentUser} matchedUser={matchedUser} />
  );
  const tree = component.toJSON();
  expect(tree).toMatchSnapshot();
});
