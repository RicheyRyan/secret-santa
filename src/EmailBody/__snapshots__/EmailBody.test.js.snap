exports[`test Cta component matches snapshot 1`] = `
<div
  className="email-body">
  <p>
    Hi 
    Test
    ,
  </p>
  <p>
    Welcome to Secret Santa!
  </p>
  <p>
    You\'ve been chosen to get a gift for 
    Test Name
    .
  </p>
  <p>
    Make sure you get them something nice.
  </p>
  <p>
    All the best,
  </p>
  <p>
    The Secret Santa Team
  </p>
</div>
`;
